apps/frontend/src/components/Search.tsx

```
@@ -0,0 +1,54 @@
+          exit={{ width: 0, opacity: 0 }}
+          onSubmit={handleSearch}
+          className="flex items-center gap-2 bg-gray-100 rounded-lg px-3 py-2"
+        >
+          <SearchIcon className="w-4 h-4 text-muted-foreground" />
+          <input
+            type="text"
+            value={searchQuery}
+            onChange={(e) => setSearchQuery(e.target.value)}
+            onBlur={() => !searchQuery && setIsSearchOpen(false)}
+            placeholder="Search..."
+            autoFocus
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)